sklearn/feature_selection/from_model.py

```diff
@@ -18,6 +82,8 @@ class _LearntSelectorMixin(TransformerMixin):
     ``feature_importances_`` or ``coef_`` attribute to evaluate the relative
     importance of individual features for feature selection.
     """
+    @deprecated('Support to use estimators as feature selectors will be '
```

Comment:
For example https://github.com/scikit-learn/scikit-learn/blob/0.16.0/doc/modules/feature_selection.rst#feature-selection-as-part-of-a-pipeline uses `_LearntSelectorMixin`